modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/helper/KBArticleURLHelper.java

```diff
@@ -93,7 +93,7 @@ private PortletURL _createKBAdminViewURL(KBArticle kbArticle) {
 		).setParameter(
 			"resourcePrimKey", kbArticle.getResourcePrimKey()
 		).setParameter(
-			"selectedItemId", kbArticle.getKbArticleId()
+			"selectedItemId", kbArticle.getResourcePrimKey()
```

Comment:
Why have you changed this?

have you talked with @boton about change the behavior of how we select and mark the selectedItemId on the navigation map?

Reply:
With this changes the navigation is broken:

previously

https://user-images.githubusercontent.com/47524751/195123159-0790fb77-4e48-4537-8a2e-3f24c9b5bacf.mp4


now:

https://user-images.githubusercontent.com/47524751/195122436-e6e8d2b4-33e5-4af2-9870-3b9559ce501f.mp4